resources/eventing/charts/nats/templates/00-prereqs.yaml

```diff
@@ -0,0 +1,130 @@
+{{- if .Values.install.enabled }}
```

Comment:
These are the original names from NATS repo. These files are only copied here. For now, it could be better to have the same name and structure as in the NATS repo. Later, we can decide if we want to modify them, which of course will generate some maintenance work for us...